src/main/java/gov/usgs/locaux/LocUtil.java

```diff
@@ -152,30 +152,29 @@
   public static final double DEG2KM = 6371d * Math.PI / 180d;
 
   /** An int constant representing the maximum number of iteration stages to attempt. */
-  public static final int STAGELIMIT = 2;
+  /** JMP 1/13/2022 Added a third stage due to large number of events that did not converge */
+  public static final int STAGELIMIT = 3;
 
   /** A double constant representing the initial step length to start each iteration loop with. */
   public static final double INITIALSTEPLEN = 50d;
 
   /** An array of int constants representing the maximum number of iterations for each stage. */
-  // public static final int[] ITERATIONSTAGELIMITS = {15, 20, 20, 20, 20, 20, 20, 20, 20, 20};
-  public static final int[] ITERATIONSTAGELIMITS = {
-    15, 20
-  }; // Collapse to with and without decorrelation sub-loops 9/16/19.
+  /** JMP 1/13/2022 Added a third stage due to large number of events that did not converge */
+  public static final int[] ITERATIONSTAGELIMITS = {15, 20, 30};
```

Comment:
why does it go 15, 20, 30 rather than 15, 20, 25? maybe I can tell later...

Reply:
I went to 30 because ray indicated that this did not converge might be due to not enough iterations, so I tried 30 and it  worked, I did not extensively search for an optimum number